modules/api/src/main/java/org/eclipse/xpanse/api/controllers/ServiceTemplateApi.java

```diff
@@ -324,34 +332,34 @@ public ServiceTemplateDetailVo getServiceTemplateDetailsById(
             produces = MediaType.APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.OK)
     @AuditApiRequest(methodName = "getCspFromServiceTemplateId")
-    public List<ServiceTemplateHistoryVo> getServiceTemplateHistoryByServiceTemplateId(
+    public List<ServiceTemplateRequestHistory> getServiceTemplateHistoryByServiceTemplateId(
             @Parameter(name = "serviceTemplateId", description = "id of service template")
-            @PathVariable("serviceTemplateId") String serviceTemplateId,
+            @PathVariable("serviceTemplateId") UUID serviceTemplateId,
             @Parameter(name = "requestType", description = "type of service template request")
             @RequestParam(name = "requestType", required = false)
             ServiceTemplateRequestType requestType,
             @Parameter(name = "changeStatus", description = "status of service template request")
             @RequestParam(name = "changeStatus", required = false)
-            ServiceTemplateChangeStatus changeStatus) {
+            ServiceTemplateRequestStatus changeStatus) {
```

Comment:
should we keep the argument as 'changeStatus' or make it to 'requestStatus'

Reply:
Updated.